src/ic-cdk/src/api/call.rs

```diff
@@ -355,6 +358,12 @@ fn decoder_error_to_reject<T>(err: candid::error::Error) -> (RejectionCode, Stri
     )
 }
 
+fn decode_args<T: for<'a> ArgumentDecoder<'a>>(bytes: &[u8]) -> candid::Result<T> {
+    let mut config = DecoderConfig::new();
+    config.set_skipping_quota(100_000); // should we enforce this or only for untrusted calls?
```

Comment:
I think it makes sense to have it. I wouldn't expect that an API evolves in a way that it used to expect a lot of data and now it doesn't.

Reply:
Added `call_with_config` function instead.